.gitignore

```diff
@@ -1,4 +1,4 @@
-.classpath
+ .classpath
```

Comment:
This ended up allowing adds for a whole bunch of generated `.classpath` files 😢.

Please revert this line and run `find . -name '.classpath' | xargs git rm` to remove them from being tracked.